### DISCUSSION TITLE
The process of migrating a minio cluster to other virtual machines

### DISCUSSION THREAD

**Discussion by kuznetsov-as** [minio/minio#20751] [Q&A]
Hi!
I have a minio cluster which consists of 4 nodes, one disk in each. I am using docker for my deployment. I need to migrate this cluster to new virtual machines. After studying the documentation, I have built the following order of actions for myself:

1) Start 4 new virtual machines, create a new, completely empty cluster.

2) Mirroring the buckets from the old cluster to the new one:

```
mc alias set source SOURCE_IP user password

mc alias set target TARGET_IP user password

mc mirror source target
```

3) Export and import configuration:

```
mc admin config export source > config.txt

mc admin config import target < config.txt
```

4) Do iam export and import:

```
mc admin cluster iam export source --output ./myminio-iam.zip

mc admin cluster iam import target ./myminio-iam.zip
```

5) Switch node addresses in the balancer

Please reply, do I understand the migration process from one minio cluster to another minio cluster correctly? Maybe I am missing something?

**Comment by klauspost** [comment]
Looks reasonable. Consider mirroring with `--watch`, if you are still using the old cluster while mirroring.

**Comment by kuznetsov-as** [comment]
@klauspost Unfortunately not everything went well. After mirroring and importing the configuration, I noticed that the quota configuration was not transferred from source bucket to target bucket. I use the command: **mc mirror --preserve source target** and the latest version of the minio image.
What am I missing?

**Reply by klauspost** [reply]
Mirror does not transfer any configuration, only objects.
